v4l2/v4l2_codecbase.cpp

```diff
@@ -279,12 +281,13 @@ const char* V4l2CodecBase::IoctlCommandString(int command)
 }
 #endif
 
-int32_t V4l2CodecBase::ioctl(int command, void* arg)
+int32_t V4l2CodecBase::ioctl(uint64_t command, void* arg)
```

Comment:
1. you can change vaapi_v4l2_video_device.h/cc slightly to be compatible with it:
typedef int32 (*VaapiV4L2Ioctl)(int32 fd, unsigned long cmd, ...); 
change the 'unsigned long' to 'uint64_t'.
similar some other places
2. more over, you can submit another patch to chrome on v4l2_video_device.h on
  virtual int Ioctl(int request, void* arg) = 0;

we are lucky that vaapi_v4l2 isn't broken for chrome because:
though compiler treat the ioctrl command (kernel defined macro) as 'unsigned long', its value is still inside the range of uint32_t. the conversion between 'long int' and int32_t still keeps the value part.

i found the issue when I tried printf.